components/feature/downloads/src/main/java/mozilla/components/feature/downloads/share/ShareDownloadFeature.kt

```diff
@@ -55,12 +56,6 @@ private const val OPERATION_TIMEOUT_MILLIS = 1000L
 @VisibleForTesting
 internal const val CHARACTERS_IN_SHARE_TEXT_LIMIT = 200
 
-/**
- * Coroutine dispatcher used for the cleanup of old cached files. Defaults to IO.
- */
-@VisibleForTesting
-internal var cleanupCacheCoroutineDispatcher = IO
-
```

Comment:
This issue was that this translate into static variable and depending on the order in which the tests get executed, we will have inconsistent results, as the variable is shared and modified by other tests

Reply:
![image](https://user-images.githubusercontent.com/773158/106955694-922e4200-6703-11eb-9b5c-1a3f59248798.png)